rudder-core/src/main/scala/com/normation/cfclerk/services/impl/GitTechniqueReader.scala

```diff
@@ -648,10 +648,7 @@ class GitTechniqueReader(
 
       val techniqueId = TechniqueId(policyName,policyVersion)
 
-      //check if the expected_report.csv file exists
-      val hasExpectedReportCsv = gitCheckreportingDescriptorExistence(techniqueId, revTreeId)
```

Comment:
shouldn't you drop the checkreportingDescriptorExistence and gitCheckreportingDescriptorExistence as well ?

Reply:
Probably

Reply:
And done!